src/styles/mixins.scss

```diff
@@ -0,0 +1,6 @@
+@mixin onDesktop {
```

Comment:
This file better name with '_'  in beginning and move to utils block.

More about file structure in scss in this [video, timing 1:00](https://mate.academy/learn/css-advanced/sass#/video/931)

Reply:
++ new dir created!